src/Microsoft.OpenApi.OData.Reader/Generator/OpenApiInfoGenerator.cs

```diff
@@ -93,5 +98,26 @@ private static string GetDescription(this ODataContext context)
             // so a default description should be provided if no Core.LongDescription annotation is present.
             return "This OData service is located at " + context.Settings.ServiceRoot.OriginalString;
         }
+
+        private static Dictionary<string, IOpenApiExtension> GetExtensions(this ODataContext context)
+        {
+            Debug.Assert(context != null);
+
+            if (context.Settings.ShowAssemblyInfo)
+            {
+                return new Dictionary<string, IOpenApiExtension>()
+                {
+                    {
+                        "x-generatedBy",
+                        new OpenApiObject
+                        {
+                            { "toolName", new OpenApiString("OpenAPI.NET.CSDL") },
```

Comment:
I think we should use the officially recognized nuget name for this library `Microsoft.OpenApi.OData`